web/api/netdata-swagger.json

```diff
@@ -1091,6 +1091,35 @@
               "host2.example.com"
             ]
           },
+          "mirrored_hosts_status": {
+            "type": "array",
+            "description": "List of details of hosts mirrored to this served (including self). Indexes correspod to indexes in \"mirrored_hosts\".",
```

Comment:
@underhood we have a typo error in the line `1096, instead `correspond`, you wrote `correspod`.

Reply:
Thank you, fixed
I will start an internal investigation to determine if the culprit in this case was my brain, fingers or keyboard.